app/facades/back_end_facade.rb

```diff
@@ -2,11 +2,39 @@ class BackEndFacade
   class << self
     def get_user(google_id)
       user = BackEndService.get_user(google_id)
-      user[:data][:id].present? ? User.new(user[:data]) : nil
+      user[:data].present? ? User.new(user[:data]) : nil
     end
 
     def create_user(params)
       BackEndService.create_user(params)
     end
+
+    def get_user_friends(user_id)
+      friends = BackEndService.get_friends(user_id)
+      friends[:data].map do |friend|
+        User.new(friend[:attributes])
+      end
+    end
+
+    def get_user_gyms(user_id)
+      gyms = BackEndService.get_gyms(user_id)
+      gyms[:data].map do |gym|
+        UserGym.new(gym[:attributes])
+      end
+    end
```

Comment:
We might consider changing this to `get_user_gym_memberships` and hit the `GET /api/v1/users/:id/gym_memberships` endpoint.  This would save a backend API call to the Yelp API and just call the BE database tables.  

The `GymMemberships` in the backend now store the gym's `yelp_gym_id` and `gym_name`.  Will that give us enough info for what we are trying to display on the dashboard with this method?

Reply:
[Issue](https://github.com/tvaroglu/spot_me_frontend/issues/19) opened